et.json

```diff
@@ -0,0 +1,296 @@
+        "meta" : {
+            "thForDate" : false,
+            "dateFormat" : "[date] [month] [year]",
+            "months" : [
+                "Jaanuar",
+                "Veebruar",
+                "Märts",
+                "Aprill",
+                "Mai",
+                "Juuni",
+                "Juuli",
+                "August",
+                "September",
+                "Oktoober",
+                "November",
+                "Detsember"
+            ],
+            "types" : {
+                "backpack" : "Seljakotid",
+                "emote" : "Emote",
+                "glider" : "Deltaplaan",
+                "emoji" : "Emojid",
+                "loading" : "Laadimisekraanid",
+                "outfit" : "Kostüümid",
+                "pickaxe" : "Kirka",
+                "skydive" : "Vabalangemise Jutt",
+                "umbrella" : "Vihmavari",
+                "spray" : "Pihusti",
+                "toy" : "Mänguasi",
+                "pet" : "Lemmikloom",
+                "music" : "Muusika",
+                "wrap" : "Paber",
+                "bundle" : "Kimp",
+                "misc" : "Muu"
+            },
+            "rarity" : {
+                "legendary" : "Legendaarney",
+                "epic" : "Eepiline",
+                "rare" : "haruldane",
+                "uncommon" : "Ebaharilik",
+                "common" : "Harilik"
+            },
+            "translationNotAvailable" : "Kahjuks pole siin lehel tõlkeid.",
+            "translationIncomplete" : "Kõik ei ole siin lehel tõlgitud."
+        },
+        "nav" : {
+            "home" : "kodu",
+            "shop" : "pood",
+            "cosmetics" : {
+                "dropdown" : "kosmeetika esemed",
+                "upcoming" : "tulemas",
+                "list" : "nimekiri",
+                "png" : "png",
+                "gallery" : "galerii",
+                "icons" : "ikoonid",
+                "reminders" : "meeldetuletused",
+                "history" : "poe ajalugu"
+            },
+            "news" : "uudised",
+            "modes" : "moodid",
+            "randomiser" : "randomiseerija",
+            "about" : {
+                "dropdown" : "info",
+                "translate" : "tõlkimine",
+                "about" : "info fnbr.co kohta",
+                "privacy" : "puutumatusnormide kirjeldus",
+                "donate" : "Anneta",
+                "discord" : "Discord",
+                "twitter" : "Twitter"
+            },
+            "api" : "api",
+            "account" : {
+                "dropdown" : "kasutaja",
+                "account" : "kasutaja",
+                "logout" : "logi välja"
+            },
+            "languagePicker" : {
+                "trigger" : "keel",
+                "title" : "Vali keel",
+                "currentLanguage" : "Praegu kasutate [language]",
+                "error" : "Tekkis viga keele faili tõmbamisel",
+                "authors" : "Tõlkis: MrMightyGamer#1337",
```

Comment:
Please remove your name (` MrMightyGamer#1337`). I don't think this string is currently in use, but it should automatically append the name of each author, possibly also for other languages.